R/write.magpie.R

```diff
@@ -99,7 +99,7 @@ write.magpie <- function(x, file_name, file_folder = "", file_type = NULL, appen
     mode <- as.character(777 - as.integer(as.character(umask)))
   }
   if (is.null(x)) x <- as.magpie(numeric(0))
-  if (is.magpie(x)) {
+  if (is.magpie(x) | class(x)  == "RasterBrick") {
```

Comment:
There needs to be a check somewhere that only the allowed file_types are being used when RasterBrick is chosen. Probably after line 107 which fills the file_type variable properly if it did not happen before. Or move the file_type step more to the front and then make a combined check here `(is.magpie(x) || (class(x) == "RasterBrick" && file_type %in% RasterFileTypes))`